assets/js/vue/components/research-group/PeopleTab.vue

```diff
@@ -3,12 +3,26 @@
         <b-card-text>
             <b-card class="card-product" no-body v-for="person in personResearchGroups" v-bind:key="person.id">
                 <b-card-body>
-                    <p>
-                        <strong>{{ person.person.firstName + " " + person.person.lastName }}</strong> <br> <br>
-                        {{ person.label }} <br>
-                        {{ person.person.organization }} <br>
-                        {{ person.person.emailAddress }} <br>
-                    </p>
+                    <div>
+                        <p>
+                            <strong>{{ person.person.firstName + " " + person.person.lastName }}</strong> <br> <br>
+                        </p>
+                    </div>
+                    <div>
+                        {{ person.label }}
+                    </div>
+                    <div class="text-overflow"
```

Comment:
Use: `text-truncate` (a bootstrap class) and also `mw-50` (or whatever is needed). Instead of the CSS below.
Remember, you can do _nearly_ all CSS with Bootstrap.

Reply:
For this we can't use the bootstrap width class (w-50), I tried it, it changes the whole tab layout. But i dint know about text-truncate, will use that 👍